tests/unit/test_group_project.py

```diff
@@ -0,0 +1,206 @@
+import ddt
+from unittest import TestCase
+import mock
+from group_project_v2.group_project import GroupActivityXBlock
+from xblock.runtime import Runtime
+
+
+from group_project_v2.stage_components import GroupProjectReviewQuestionXBlock
+from xblock.field_data import DictFieldData
+from tests.utils import TestWithPatchesMixin
+
+
+def _make_question(question_id):
+    question_mock = mock.create_autospec(spec=GroupProjectReviewQuestionXBlock)
+    question_mock.question_id = question_id
+    return question_mock
+
+
+def _make_reviews(reviews):
+    result = []
+    for review in reviews:
+        reviewer, question, answer = review
+        result.append({'reviewer': reviewer, 'question': question, 'answer': answer})
+    return result
```

Comment:
Alternatively:

    return [{'reviewer': reviewer, 'question': question, 'answer': answer}
            for reviewer, question, answer in reviews]